html/elements/meta.json

```
@@ -477,10 +477,17 @@
                     "notes": "Chrome reports support, but does not actually use the color anywhere."
                   }
                 ],
-                "chrome_android": {
-                  "version_added": "80",
-                  "notes": "Chrome for Android does not use the color on devices with native dark mode enabled."
-                },
+                "chrome_android": [
+                  {
+                    "version_added": "92"
+                  },
+                  {
+                    "version_added": "39",
+                    "version_removed": "91",
```

Note that `version_removed` is the version after `version_last`, i.e. the version in which the support changed. Was this feature really not present in Chrome 91? Otherwise:

```suggestion
                    "version_removed": "92",
```

PS: The Chrome statements seem to have the same issue.

---

Oh good- I based this off of the Chrome statement, and had the thought that an inclusive `version_removed` was likely to cause issues, so I'm glad to hear that isn't correct. 

I fixed this, and updated the Chrome statement as well, since that is incorrect.